css/button.css

```diff
@@ -0,0 +1,42 @@
+:root {
+    --primaryColor: #414141;
+    --secondaryColor: #2b7a78;
+    --lightGrey: #dddddd;
+    --buttonColor: #17252a;
```

Comment:
add color name not specific to component

Reply:
done